docs/user/tutorials/install_epinio_on_gke.md

```diff
@@ -41,9 +41,9 @@ The output of the command will print the `loadbalancer-ip`.
 #### Install Epinio in cluster
```

Comment:
```suggestion
#### Install Epinio In Cluster
```

Like most other changes to titles ?

Reply:
Yes, I thought we could just keep level 4 headlines as sentences, because the font will have a similar size ;)
Better to be consistent though.